platform/platform-configuration-listener/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -22,6 +22,7 @@
             <ext:property name="configFileDirectory" value="${ddf.home}/etc"/>
             <ext:property name="processedDirectory" value="${ddf.home}/etc/processed"/>
             <ext:property name="failedDirectory" value="${ddf.home}/etc/failed"/>
+            <ext:property name="exportedDirectory" value="${ddf.home}/etc/exported"/>
```

Comment:
Now that `export()` takes in the directory path, we should be able to remove it from this blueprint file and move to the command blueprint file instead.